spec/models/instance/settings/effective_settings_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+RSpec.describe Instance::Settings::EffectiveSettings, type: :model do
+  let(:settings) do
+    mock_settings(components: {
+                    A: { enabled: true }
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.